backend/fms_core/services/experiment_run.py

```diff
@@ -20,8 +20,16 @@ def create_experiment_run(experiment_run_name,
     errors = []
     warnings = []
 
+    if run_type_obj is None:
+        errors.append('Run type is required to create an experiment run.')
+    if process_properties is None:
+        errors.append('Process properties are required to create an experiment run.')
+
+    if errors:
+        return None, errors, warnings
+
     if not protocols_dict:
-        protocols_dict = run_type_obj.get_protocols_dict()
+        protocols_dict = run_type_obj.get_protocols_dict
```

Comment:
I would tend to instead remove the `@property` decorator instead. This is not really a property.